linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages_and_set_runlevel/ansible/shared.yml

```diff
@@ -0,0 +1,25 @@
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_ol
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Ensure xorg packages are removed
+  package:
+    name: "{{ item }}"
+    state: absent
+  loop:
```

Comment:
I would urge against using loops in Ansible yum package modules.. Instead, you should opt for passing in to name for efficiencies:
```
- name: Ensure xorg packages are removed
  package:
    name:
      - xorg-x11-server-Xorg
      - xorg-x11-server-common
      - xorg-x11-server-utils
{{% if product not in ["rhel7", "ol7"] %}}
      - xorg-x11-server-Xwayland
{{% endif %}}
    state: absent
```
see: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#notes